FirebaseABTesting/Sources/Public/FirebaseABTesting/FIRLifecycleEvents.h

```diff
@@ -19,13 +19,19 @@ NS_ASSUME_NONNULL_BEGIN
 /// Default event name for when an experiment is set.
 extern NSString *const FIRSetExperimentEventName NS_SWIFT_NAME(DefaultSetExperimentEventName);
 /// Default event name for when an experiment is activated.
+// clang-format off
+// clang-format12 merges the next two lines.
```

Comment:
Nit: since these are back to back, you could mention it doesn't honour line length for `NS_SWIFT_NAME` and leave it off all the way until line 34